adoc/Private-Registry-docinfo.xml

```diff
@@ -0,0 +1,87 @@
+<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+    <dm:bugtracker>
+        <dm:url>https://github.com/SUSE/suse-best-practices/issues/new</dm:url>
+        <dm:product>SUSE Private Registry Powered by Harbor 2.1</dm:product>
+    </dm:bugtracker>
+    <dm:editurl>https://github.com/SUSE/suse-best-practices/blob/master/adoc/Harbor-Private-Registry.adoc</dm:editurl>
+</dm:docmanager>
+
+
+<!--<title>SUSE(R) Enterprise Storage Implementation Guide for Veeam Backup & Recovery</title>-->
```

Comment:
whats this?

Reply:
Just some leftover artifacts from the file I used as a template for this metadata info. I'll remove the comments.